packages/phone-number-privacy/signer/src/domain/domain.service.ts

```diff
@@ -139,6 +141,11 @@ export class DomainService implements IDomainService {
       hash: domainHash(domain),
     })
 
+    const key: Key = {
+      name: KeyName.domains,
+      version: Number(request.headers.keyVersion) || config.keystore.keys.domains.latest,
```

Comment:
Suggestions to adjust the header name and to use `??` instead of `||`, which prevents the case of `0` from being handled as `undefined`. I'm also not cure what happens with `NaN` in the current code.
```suggestion
      version: Number(request.headers['Odis-Key-Version'] ?? config.keystore.keys.domains.latest),
```